docs/technologies/languages/nodejs.mdx

```diff
@@ -96,4 +96,42 @@ If your project uses the [Yarn PnP](https://yarnpkg.com/features/pnp) installati
 
 ```yml title=".yarnrc"
 enableGlobalCache: false
-```
\ No newline at end of file
+```
+
+## Environment Variables
+
+In der Regel werden Umgebungsvariablen genutzt, um Node.js-Apps zu konfigurieren. Je nach Node.js-Version empfehlen wir unterschiedliche Varianten, um diese zu definieren.
```

Comment:
In english, please. 🇬🇧   Die deutschen Files sind im `i18n`-Ordner. 🙂

Reply:
Ich weiiiiß 😄 - schiebe ich gleich nach, Dave liest gerade einmal drüber.

Reply:
Ist nun auch auf Englisch eingetragen. Magst du bitte nochmal drüberschauen?